multicast/env.py

```
@@ -98,6 +99,57 @@
+		>>> validate_buffer_size("1316")  # String that can be converted
+		True
+		>>> try:
+		...     validate_buffer_size('invalid')
+		... except ValueError:
+		...     print('ValueError raised')
+		ValueError raised
+
+	"""
+	try:
+		size_num = int(size)
+		return 0 < size_num <= 65507
```

Ruff (0.8.2)

Consider moving this statement to an `else` block

(TRY300)